src/Components/Views/Survey/survey.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react'
+import styled from 'react-emotion'
+import { Shades, Colors } from 'Elements/Colors/colors'
+import { StyledButton, buttonReset } from 'Elements/Buttons/button'
+import { PanelBase } from 'Elements/Foundations/foundation'
+
+const PanelWrapper = styled('div')`
+  ${PanelBase};
+  padding: 0.8rem;
+  text-align: center;
+`
+const CancelSurvey = styled('button')`
+  ${buttonReset};
+  font-size: 0.6rem;
+  color: ${Shades.overcast};
+  padding-top: 0.6rem;
+  &:hover {
+    color: ${Colors.teal};
+  }
+`
+
+const FieldSet = styled('fieldset')`
+  all: unset;
+`
+const FieldTitle = styled('legend')`
+  all: unset;
+  font-size: 1rem;
+  padding: 0.2rem 3.5rem 1rem;
+  font-weight: 600;
+`
+const FieldOption = styled('button')`
+  ${StyledButton};
+  width: 100%;
+  margin-bottom: 4px;
+  font-size: 0.875rem;
+`
+
+function FieldOptions({ options, name }) {
+  return options.map(option => {
+    const { text, value } = option
+    return (
+      <FieldOption onClick={() => console.log(`${name}:${value}`)}>
+        {text}
+      </FieldOption>
+    )
+  })
+}
+
+function Fields({ fields }) {
+  return fields.map(fieldset => {
+    const { label, options, name } = fieldset
+    return (
+      <FieldSet>
```

Comment:
keys required for FieldSet items

<img width="820" alt="screen shot 2018-07-16 at 1 27 27 pm" src="https://user-images.githubusercontent.com/1546207/42782047-929dbbcc-88fc-11e8-8272-61b3b6467441.png">
<img width="580" alt="screen shot 2018-07-16 at 1 30 48 pm" src="https://user-images.githubusercontent.com/1546207/42782045-9270c4e6-88fc-11e8-8ccc-2f38e0290308.png">
<img width="567" alt="screen shot 2018-07-16 at 1 30 40 pm" src="https://user-images.githubusercontent.com/1546207/42782046-9285c684-88fc-11e8-8e70-1e733b39273a.png">

Reply:
Thanks!  I will update those two key issues.